wasm-host/src/vm_wamr.rs

```diff
@@ -55,17 +56,17 @@ pub fn run_func(wasm_file: String, func_name: &str, gas_cap: Option<u32>, data_s
         .register_host_function("get_nft", get_nft as *mut c_void, "(*~*~*~)i", 1000, data_provider.as_ptr())
         .register_host_function("trace", trace as *mut c_void, "(*~*~i)i", 500, data_provider.as_ptr())
         .register_host_function("trace_num", trace_num as *mut c_void, "(*~I)i", 500, data_provider.as_ptr())
-        .register_host_function("float_from_int", float_from_int as *mut c_void, "(I*~i)i", 100, data_provider.as_ptr())
-        .register_host_function("float_from_uint", float_from_uint as *mut c_void, "(*~*~i)i", 100, data_provider.as_ptr())
-        .register_host_function("float_set", float_set as *mut c_void, "(iI*~i)i", 100, data_provider.as_ptr())
-        .register_host_function("float_compare", float_compare as *mut c_void, "(*~*~)i", 100, data_provider.as_ptr())
-        .register_host_function("float_add", float_add as *mut c_void, "(*~*~*~i)i", 100, data_provider.as_ptr())
-        .register_host_function("float_subtract", float_subtract as *mut c_void, "(*~*~*~i)i", 100, data_provider.as_ptr())
-        .register_host_function("float_multiply", float_multiply as *mut c_void, "(*~*~*~i)i", 100, data_provider.as_ptr())
-        .register_host_function("float_divide", float_divide as *mut c_void, "(*~*~*~i)i", 100, data_provider.as_ptr())
-        .register_host_function("float_pow", float_pow as *mut c_void, "(*~i*~i)i", 100, data_provider.as_ptr())
-        .register_host_function("float_root", float_root as *mut c_void, "(*~i*~i)i", 100, data_provider.as_ptr())
-        .register_host_function("float_log", float_log as *mut c_void, "(*~*~i)i", 100, data_provider.as_ptr())
+        .register_host_function("float_from_int", iou_value_from_int as *mut c_void, "(I*~i)i", 1000, data_provider.as_ptr())
```

Comment:
Per our discussion on zoom, gentle nudge to create an issue to track renaming these back to use `float`